backend/stackexchange/test_stack_api.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+from backend.stackexchange.stack_api import ApiResponse, User, from_dict
+
+
+class TestStackApi(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        cls.user_json_1 = ApiResponse('stackoverflow', '2335489')
+        cls.user_json_2 = ApiResponse('stackoverflow', '8078494')
+        cls.user_1 = from_dict(data_class=User, data=cls.user_json_1.data_specific_values)
```

Comment:
As I understand, from_dict is a function for parse json to as some class. So, we need to use it from the repository perspective, not test.